src/pybind/rgw/setup.py

```diff
@@ -157,7 +192,24 @@ def cythonize(x, **kwargs):
 
 flags = get_python_flags()
 
-setup(
+cythized = cythonize(
```

Comment:
why is this necessary? or put in other words, why shall we run cythonize twice?

Reply:
It's not. It was a leftover from when I was exploring and trying to figure out where the flags kept being reintroduced. I'll pull it out.